kelpie-f446/Core/Modules/AppendageActuationModule.h

```diff
@@ -2,12 +2,22 @@
  * AppendageActuationModule.h
  *
  *  Created on: Oct 22, 2022
- *      Author: mingy
+ *      Author: mingy, eric
  */
 
 #ifndef MODULES_APPENDAGEACTUATIONMODULE_H_
 #define MODULES_APPENDAGEACTUATIONMODULE_H_
 
+#include "UserTypes.h"
+#include "stm32f4xx_hal.h"
+#include "ServoDriver.h"
 
+typedef enum {
+	CLAW_OPEN = -1,
```

Comment:
0xFF is normally used instead of -1 since most numbers are unsigned. not a big issue